dbms/src/Storages/KVStore/Read/LearnerReadWorker.cpp

```diff
@@ -265,7 +265,7 @@ void LearnerReadWorker::recordReadIndexError(
                 LOG_DEBUG(
                     log,
                     "meet abnormal region error {}, [region_id={}]",
-                    resp.region_error().DebugString(),
+                    resp.region_error().GetTypeName(),
```

Comment:
I think `DebugString` contains the infomation of `GetTypeName`? Why we need to change it.

Reply:
Because I found it is empty